### ISSUE TITLE
Integration tests

### ISSUE THREAD

**Issue by vovacha** [playzero/zero-network#113]
**Questions:**
- Do we want to apply the integration test from UI side or check every component: graph, UI, chain

**Test groups:**
- UI tests
- Chain tests
- Graph tests

**Selenium, Puppeteer, etc. scenarios:**
- create org from UI (or js scaffolder)
- check if org was created on the org page
...

**Comment by 2075** [comment]
yes, currently inan did it manually, but that's not very efficient. i think there was a repo for automating this on gamedao repos

**Comment by vovacha** [comment]
Let's discuss it later. I have not included this issue in the Milestone.
